src/app/categories/[categoryId]/pairwise-ranking/page.tsx

```diff
@@ -63,11 +64,17 @@ const CategoryPairwiseRankingPage = () => {
 	const totalPairs = pairwisePairs?.data.totalPairs ?? 1;
 	let progressPercentage = 0;
 
+	
+
 	if (totalPairs !== 0) {
 		progressPercentage = (votedPairs / totalPairs / threshold) * 100;
 	}
 
 	const handleVote = async (pickedId: number) => {
+		posthog.capture('Comparing categories',{categories:[firstProject.name,secondProject.name]})
```

Comment:
The filename is a little misleading but here we're actually comparing projects.

We do category pairwise comparisons here: `src/app/category-ranking/page.tsx`